cmd/markdown/atmos_cast_render_usage.md

```
@@ -0,0 +1,8 @@
+- Render an asciicast recording
+
+```shell
+ $ atmos cast render demo.cast --output=demo.gif
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)